improver_tests/wxcode/wxcode/test_ModalCode.py

```diff
@@ -164,12 +196,13 @@ def test_metadata(wxcode_series):
     def as_utc_timestamp(time):
         return timegm(time.utctimetuple())
 
-    model_id_attr, wxcode_cubes = wxcode_series
+    model_id_attr, record_run_attr, wxcode_cubes = wxcode_series
 
+    kwargs = {}
     if model_id_attr:
-        kwargs = {"model_id_attr": MODEL_ID_ATTR}
-    else:
-        kwargs = {}
+        kwargs.update({"model_id_attr": MODEL_ID_ATTR})
+    if record_run_attr:
```

Comment:
I don't think you're checking that anything changes in the output cube if this is set.

Reply:
Yep, thanks for spotting that. Now covered.